international-labour-organization.csl

```diff
@@ -137,16 +144,20 @@
         <date-part name="year"/>
       </date>
       <choose>
-        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <if type="book" match="any">
           <group prefix=" " delimiter=". " suffix=".">
-            <text macro="title"/>
+            <text macro="title" font-style="italic"/>
+            <text variable="collection-title"/>
             <text macro="edition"/>
             <text macro="editor"/>
+            <text variable="volume" form="short" text-case="capitalize-first"/>
```

Comment:
this seems odd - while it's valid CSL, I don't think any reference managers offers a short for of the volume variable. Do you mean to include a label here that should be short, i.e. Vol.? You'd have to set that via a label or a term.